Gaijinizer patch/Map191.txt

```diff
@@ -396,9 +430,8 @@ release of magic.#MTLed
 天使の魔力を『暴走』させるには
 あまりに大きすぎたのだ。
 > CONTEXT: Map191/events/1/pages/0/293/Dialogue/other2
-Well, \\N[1]-san's bowl of magic is
-To "run away" the angel's magic
-It was too big.#MTLed
+And \\N[1]-kun's vessel of magic was too big
+to force the magic to go out of control.#RonIffy
 > END STRING
```

Comment:
I think this flows a bit better:
>_**Additionally, \\\N[1]-kun's natural capacity
to store the magic was too great to force it
 to go out of control.**_